boards/samr21-xpro/include/board.h

```diff
@@ -48,6 +48,13 @@ extern "C" {
 #define AT86RF231_SLEEP    GPIO_7
 
 #define AT86RF231_SPI_SPEED SPI_SPEED_1MHZ
+
+/*Thread optimization for samr21*/
+#define RADIO_STACK_SIZE                (1024)
+#define RPL_PROCESS_STACKSIZE           (1024)
+
+/*samr21 is very memory constrained and does not run with the default RPL_MAX_ROUTING_ENTRIES (= 128)*/
+#define RPL_MAX_ROUTING_ENTRIES            (8)
```

Comment:
Yes, the one in this header is defined after the command line option, resulting in:
```
./RIOT/boards/samr21-xpro/include/board.h:57:0: warning: "RPL_MAX_ROUTING_ENTRIES" redefined
 #define RPL_MAX_ROUTING_ENTRIES            (8)
```
so I vote to change it to:
```
#ifndef RPL_MAX_ROUTING_ENTRIES
#   define RPL_MAX_ROUTING_ENTRIES            (8)
#endif
```